iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/serviceclient/RegistryManagerTests.java

```diff
@@ -459,35 +459,43 @@ public void apply_configuration_e2e() throws Exception
     public void deviceCreationWithDeviceScope() throws IOException, InterruptedException, IotHubException, URISyntaxException
     {
         // Arrange
-        RegistryManagerTestInstance testInstance = new RegistryManagerTestInstance();
-        deleteDeviceIfItExistsAlready(testInstance.registryManager, testInstance.deviceId);
+        RegistryManagerTestInstance edge1Instance = new RegistryManagerTestInstance();
```

Comment:
Is it okay to have more than 1 instance of this? It gets me unique device Ids, but also duplicates the registry manager client.

Reply:
Typically a single test has one test instance that contains all the relevant deviceIds and clients. In the case of your test, I'd forgo creating a test test instance per device, and just create the devices directly.

Reply:
okay, how's this?